integration-tests/tests/singlestore/singlestore-common.ts

```diff
@@ -2111,45 +2111,6 @@ export function tests(driver?: string) {
 			await db.execute(sql`drop table ${products}`);
 		});
 
-		test('transaction with options (set isolationLevel)', async (ctx) => {
-			const { db } = ctx.singlestore;
-
-			const users = singlestoreTable('users_transactions', {
-				id: serial('id').primaryKey(),
-				balance: int('balance').notNull(),
-			});
-			const products = singlestoreTable('products_transactions', {
-				id: serial('id').primaryKey(),
-				price: int('price').notNull(),
-				stock: int('stock').notNull(),
-			});
-
-			await db.execute(sql`drop table if exists ${users}`);
-			await db.execute(sql`drop table if exists ${products}`);
-
-			await db.execute(sql`create table users_transactions (id serial not null primary key, balance int not null)`);
-			await db.execute(
-				sql`create table products_transactions (id serial not null primary key, price int not null, stock int not null)`,
-			);
-
-			const [{ insertId: userId }] = await db.insert(users).values({ id: 1, balance: 100 });
-			const user = await db.select().from(users).where(eq(users.id, userId)).then((rows) => rows[0]!);
-			const [{ insertId: productId }] = await db.insert(products).values({ id: 1, price: 10, stock: 10 });
-			const product = await db.select().from(products).where(eq(products.id, productId)).then((rows) => rows[0]!);
-
-			await db.transaction(async (tx) => {
-				await tx.update(users).set({ balance: user.balance - product.price }).where(eq(users.id, user.id));
-				await tx.update(products).set({ stock: product.stock - 1 }).where(eq(products.id, product.id));
-			}, { isolationLevel: 'serializable' });
```

Comment:
SingleStore still supports `READ COMMITED` isolation level so we should test this isolation level

Reply:
I believe it's a syntax issue since the error that the engine outputs is related to `set transaction` and SingleStore doesn't support those words to start a transaction

Reply:
Nevermind, I figured that SingleStore only supports the `READ COMMITTED` isolation level and it's the default isolation level for all transactions so I decided to remove the `isolationLevel` transaction configuration param from the S2 transaction class